tests/unit/amo/components/TestAddonReviewCard.js

```diff
@@ -296,17 +348,131 @@ describe(__filename, () => {
     );
   });
 
+  it('begins deleting when clicking a delete link for a review', () => {
+    const review = signInAndDispatchSavedReview();
+    const dispatchSpy = sinon.spy(store, 'dispatch');
+    const root = render({ review, smallCard: true });
+
+    const deleteLink = renderControls(root).find('.AddonReviewCard-delete');
+    expect(deleteLink).toHaveLength(1);
+
+    const fakeEvent = createFakeEvent();
+    deleteLink.simulate('click', fakeEvent);
+
+    sinon.assert.calledWith(
+      dispatchSpy,
+      beginDeleteAddonReview({ reviewId: review.id }),
+    );
+    sinon.assert.called(fakeEvent.preventDefault);
+  });
+
+  it('renders generic delete confirmation buttons', () => {
```

Comment:
Nit: This is a general comment about a the tests that follow. I find it a bit hard to grasp what these are testing without looking closely at the code, which I suppose should maybe be expected. The issue is that there are tests for `smallCard: true` and tests for `smallCard: false` but they're not labelled that way. It seems like the tests for `smallCard: false` are stated like they're the default behaviour (e.g., `renders generic delete confirmation buttons` and `renders a confirmation prompt for deleting a review`) whereas the tests where `smallCard: true` are sometimes specifically called out and sometimes not (e.g., `renders delete confirmation buttons for a rating` and `does not render a confirmation message for smallCard=true`). I wonder if the tests might be easier to understand if you divided these into two `describe` blocks, one for `smallCard: true` and one for `smallCard: false`?

I know you said you considered adding some `describe` blocks and decided it would make things more confusing, but maybe this would be an improvement?